DataScience/Experimentation.py

```diff
@@ -406,24 +420,32 @@ def main(args):
             best_command = results[0]
             best_commands.append(['Hyper2', results[0]])
 
-        # TODO: Repeat above process of tuning parameters and interactions until convergence / no more improvements.
-
     t1 = datetime.now()
     print("\n\n*************************")
     Logger.info("Best parameters found after {}:".format((t1-t0)-timedelta(microseconds=(t1-t0).microseconds)))
     best_command.prints()
     print("*************************")
 
+    # TODO: Repeat above process of tuning parameters and interactions until convergence / no more improvements.
```

Comment:
Is this relevant here?

Reply:
That line was there from earlier. I did not add it. I think the TODO was left behind to add more ways to determine best models but we never got around to adding more hyperparameter tuning methods. Left it as is since I was not touching that code.